zulip_bots/zulip_bots/lib.py

```diff
@@ -156,6 +158,12 @@ def run_message_handler_for_bot(lib_module, quiet, config_file, bot_name):
 
     state_handler = StateHandler()
 
+    # Look in bot class to see if empty messages will be handled in this function
+    try:
+        handle_empty_here = not (lib_module.handler_class.ACCEPT_EMPTY_MESSAGES)
+    except AttributeError:
+        handle_empty_here = True
```

Comment:
Maybe the exception approach is clearer here?
- Take inverse of the found variable
- ...or default to True